### PR TITLE
Support Fedora 23

### PR DESCRIPTION
**Pull request by mvollmer**

### CODE REVIEW

**Comment by mvollmer** [comment]
Image seems to build OK, but installation of Cockpit rpms fails silently.

**Comment by mvollmer** [comment]
> Image seems to build OK, but installation of Cockpit rpms fails silently.

firewall-cmd is broken with setenforce 1, let's just switch off selinux altogether.

**Comment by mvollmer** [comment]
Ok, tests are running and failing as expected.  I'd say this is good to be merged.

**Comment by stefwalter** [comment]
@mvollmer Could you rebase this on master so the critical failures cause a backtrace? Then I can look at hubbot and retrieve the backtraces.

**Comment by mvollmer** [comment]
Rebased.

**Comment by stefwalter** [comment]
@mvollmer This needs a rebase. In addition, the following lines need to be added to the f23 setup script:

```
echo "kernel.core_pattern=|/usr/lib/systemd/systemd-coredump %p %u %g %s %t %e" > /etc/sysctl.d/50-coredump.conf
printf "[Coredump]\nStorage=journal\n" > /etc/systemd/coredump.conf
```

**Comment by stefwalter** [comment]
This is still using the old glib2: DEBUG util.py:388:  Package glib2-devel-2.45.4-1.fc23.x86_64 is already installed, skipping.

Still blocked.

**Comment by stefwalter** [comment]
Needs a rebase, both for code, and also because some of those build failures are fixed in git master.

**Comment by mvollmer** [comment]
Rebased.  Image creation hang at "docker pull" when running this locally, but the storage tests are passing now, more or less.

**Comment by mvollmer** [comment]
We don't have good control over the build environment.  I did a manual "make-rpms --clean" to cause the fixed glib2 to be installed.  Maybe we should just drop the non-clean optimization, it only saves one or two minutes.

**Comment by mvollmer** [comment]
`docker pull busybox` seems to hang.

**Comment by mvollmer** [comment]
> docker pull busybox seems to hang.

This is reproducible always in a stock Fedora 23: https://bugzilla.redhat.com/show_bug.cgi?id=1262796

Not sure what to do about it... should we put a timeout on the docker pull so that only the kube tests fail and we can run the rest?

**Comment by mvollmer** [comment]
@sub-mod, you might be interested in this.

**Comment by stefwalter** [comment]
@mvollmer Needs rebasing when we're ready to rebuild this (blocking on new docker build)?

File: test/guest/cockpit-fedora-23.setup [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+# the rest. This helps avoid things like:
+#
+#     https://bugzilla.redhat.com/show_bug.cgi?id=1211547
+#
+dnf -y upgrade dnf
+
+dnf -y upgrade
+dnf -y install $TEST_PACKAGES $COCKPIT_DEPS $IPA_CLIENT_PACKAGES
+
+rm -rf /var/log/journal/*
```
Comment (dperpeet) [inline test/guest/cockpit-fedora-23.setup]: this needs to move to the end of setup like for the other configurations in #2758

**Comment by mvollmer** [comment]
> Needs rebasing when we're ready to rebuild this (blocking on new docker build)?

Yes, let's wait for that.

File: test/guest/cockpit-fedora-23.setup (hunk 1 above)
Comment (mvollmer) [inline test/guest/cockpit-fedora-23.setup]: Done.

**Comment by mvollmer** [comment]
Rebased and removed docker workaround.

**Comment by mvollmer** [comment]
> Yes, let's wait for that.

Can be tracked here: https://bugzilla.redhat.com/show_bug.cgi?id=1262796

**Comment by stefwalter** [comment]
F22 failure is #2858

**Pull request closed without merging**